aoc-client/src/lib.rs

```diff
@@ -413,13 +414,25 @@ impl AocClient {
         Ok(calendar)
     }
 
+    fn colour_map(anns: &[RichAnnotation], s: &str) -> String {
+        for ann in anns.iter().rev() {
+            match ann {
+                RichAnnotation::Colour(c) => {
+                    return s.truecolor(c.r, c.g, c.b).to_string();
+                }
+                _ => {}
+            }
+        }
+        s.into()
+    }
+
     pub fn show_calendar(&self) -> AocResult<()> {
         let calendar_html = self.get_calendar_html()?;
-        let calendar_text = from_read_with_decorator(
+        let calendar_text = from_read_coloured(
```

Comment:
I know this is probably just a quick hack for testing but ideally we'd want this behind an optional colour flag

Reply:
We could even make coloured output as the default with a flag to revert to uncoloured.

Reply:
I've added a --no-colour-calendar option.  Happy to rename etc. to anything you prefer.